### ISSUE TITLE
Test dataset distribution

### ISSUE THREAD

**Issue by jonathantanu** [okkyibrohim/id-multi-label-hate-speech-and-abusive-language-detection#2]
Hi! I would like to know how you divide the dataset into train, validate, and test dataset in your paper here https://aclanthology.org/W19-3506.pdf thanks!

**Comment by okkyibrohim** [comment]
Hi, thanks for the questions!
As written in the paper, we use a 10-fold cross-validation strategy so that we don't separate it into train, validation, and test set manually. In this case, we use Scikit-Learn Library to implement the 10-fold cross-validation strategy, you can use that to re-implement in case you want an apple-to-apple comparison with my paper result :)
